src/angular-core/forms/form-field-control.ts

```diff
@@ -23,14 +24,26 @@ export abstract class SbbFormFieldControl<TValue> {
   readonly disabled: boolean;
   /** Whether the control is in an error state. */
   readonly errorState: boolean;
+  /**
+   * An optional name for the control type that can be used to distinguish `mat-form-field` elements
```

Comment:
introducing the next "mat" ;-)
```suggestion
   * An optional name for the control type that can be used to distinguish `sbb-form-field` elements
```